README.md

```diff
@@ -30,7 +30,7 @@ bundle yourself:
 
 return [
     // ...
-    Norzechowicz\AceEditorBundle\NorzechowiczAceEditorBundle::class => ['all' => true],
+    NorbertTech\AceEditorBundle\NorbertTechAceEditorBundle::class => ['all' => true],
```

Comment:
I would move it under `AceEditorBundle\AceEditorBundle::class` - this way it becomes completely agnostic and unrelated to my name.

Reply:
fair enough, will do.